src/edge_utils.c

```diff
@@ -2946,6 +2970,17 @@ int run_edge_loop (n2n_edge_t *eee) {
                        HASH_COUNT(eee->known_peers));
         }
 
+        if ((eee->conf.allow_routing) && (now - last_purge_host > SWEEP_TIME)) {
+            struct host_info *host, *host_tmp;
+            HASH_ITER(hh, eee->known_hosts, host, host_tmp) {
+                if (now - host->last_seen > 300) {         
```

Comment:
Please use a `#define` in `include/n2n_defines.h` instead of a hard-coded constant time out.

Reply:
HOSTINFO_AGEOUT_TIMEOUT is defined for hostinfo ageout